handlers/download.go

```diff
@@ -62,6 +93,33 @@ func (s Server) entryGet() http.HandlerFunc {
 	}
 }
 
+func renderPassphrasePrompt(w http.ResponseWriter, r *http.Request, id picoshare.EntryID, meta picoshare.UploadMetadata) {
+	renderPassphrasePromptWithError(w, r, id, meta, "")
+}
+
+func renderPassphrasePromptWithError(w http.ResponseWriter, r *http.Request, id picoshare.EntryID, meta picoshare.UploadMetadata, errorMsg string) {
+	// Minimal inline template to avoid adding static assets yet.
```

Comment:
We don't actually want to do this because it means this page is a complete mismatch with the site style.

We should define a page using the same pattern we do with everything else in `handlers/templates/pages`.